src/process/process.go

```diff
@@ -57,8 +57,8 @@ func InitAllProcs() (map[int32]*Process, error) {
 	return processes, nil
 }
 
-func NewProcess(pid int32) (*Process, error) {
-	process, err := proc.NewProcess(pid)
+func NewProcess(pid uint32) (*Process, error) {
+	process, err := proc.NewProcess(int32(pid))
```

Comment:
I completely forgot that `NewProcess()` takes an `int32`. Since that is the case, you can leave the type of `pid` to be `int32` itself. My apologies for the confusion and the default value of the `pid` can be set to `-1` again.